app/rules/page.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+
+import Footer from "@/components/footer/footer";
+import Navbar from "@/components/navbar";
+import { formatHourTo12Hour, hackathonDateInfo } from "@/lib/dates";
+
+function Rules() {
+  return (
+    <div className="flex flex-col justify-center items-center min-h-screen">
```

Comment:
Add an `id` parameter for the section for better screen reader/accessibility support.
```suggestion
    <div id="rules" className="flex flex-col justify-center items-center min-h-screen">
```